pkg/machines/actions.es6

```diff
@@ -128,6 +143,7 @@ export function updateOrAddVm({ id, name, connectionName,
     cpuModel,
     bootOrder,
     displays,
+    usagePolling,
```

Comment:
Please introduce new updateVm({ id, name, connectionName, usagePolling }) action creator to ensure the VM is not re-created if already removed.